### PR TITLE
Add params to connect to GHE

### PR DESCRIPTION
**Pull request by JbIPS**
I added a host param to allow a connection to another GitHub API than github.com, like a GitHub Entreprise API.

I let github.com as default.

### CODE REVIEW

**Comment by JbIPS** [comment]
@henrahmagix Hi, do you think you could merge this one soon? If you  need anything, just ask me

**Comment by henrahmagix** [comment]
Sorry I didn't see this @JbIPS! Please add tests and I'll merge =)

**Comment by JbIPS** [comment]
I added some type check for the new parameter and the according tests because that doesn't change the rest of the library behavior.

File: lib/github-oauth-prompt.js [hunk 1]
```diff
@@ -210,6 +209,11 @@ function getOptions (userOptions) {
         throw new Error('Option name must be a non-empty string');
     }
 
+    // Option: host.
+    if (options.host !== undefined && !isNonEmptyString(options.host)) {
```
Comment (henrahmagix) [inline lib/github-oauth-prompt.js]: You don't need to check `undefined` because `isNonEmptyString` accounts for that.

Comment (JbIPS) [inline lib/github-oauth-prompt.js]: Sure, but we need to check the empty string only if the parameter is present. If we don't, you'll get an error when you don't specify the host. For the other params, you do an undefined check and an assignation, here you only have the check.

**Comment by JbIPS** [comment]
Is this OK for you?

**Comment by JbIPS** [comment]
Little bump here :)

**Comment by henrahmagix** [comment]
:+1: Sorry for missing the notification in September

**Comment by henrahmagix** [comment]
[`v1.1.0`](https://github.com/henrahmagix/github-oauth-prompt/releases/tag/v1.1.0) released. Thanks!

**Pull request merged**